src/slic3r/GUI/GLTexture.cpp

```diff
@@ -492,8 +492,11 @@ bool GLTexture::generate_from_text(const std::string &text_str, wxFont &font, wx
 
     m_original_width = (int)w;
     m_original_height = (int)h;
-    m_width = (int)next_highest_power_of_2((uint32_t)w);
-    m_height = (int)next_highest_power_of_2((uint32_t)h);
+    //m_width  = (int)next_highest_power_of_2((uint32_t)w);
+    //m_height = (int)next_highest_power_of_2((uint32_t)h);
+    // using next_highest_power_of_2 changes ratio when height / width values dont match
+    m_width  = static_cast<int>(std::ceil(w)); 
+    m_height = static_cast<int>(std::ceil(h));
```

Comment:
This could cause problems on some GPUs, which requires texture sizes to be power of 2.